lib/quark/compose.ex

```diff
@@ -73,7 +73,7 @@ defmodule Quark.Compose do
   def g <|> f, do: compose(g, f)
 
   @doc ~S"""
-  Function composition, from the back of the lift to the front
+  Function composition, from the back of the list to the front
```

Comment:
Ah ok yes, so perhaps "lift" was not a typo after all. I've not heard the term used in that context, but that doesn't mean its not a valid statement.